src/unix/index_unix.ml

```diff
@@ -359,6 +388,10 @@ module IO : Index.IO = struct
     | None -> err_rw_lock path
 
   let unlock fd = Unix.close fd
+
+  let incr_nb_merge () = stats.nb_merge <- succ stats.nb_merge
```

Comment:
as the merges are not done by this file, `nb_merge` and `nb_replace` should probably be moved into a new `stats` value defined in metrics.ml.

Not that `Metric_unix.ml` can include the contents of that new stats value too.